person.rb

```diff
@@ -0,0 +1,33 @@
+require_relative './nameable'
+require_relative './rental'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..9999)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    super()
+  end
```

Comment:
- [ ] Like the previous CR already mentioned When the class is initialized there should be an empty rentals array (@rentals = []) created as well, after doing so your constructor should look something like:
```
  def initialize(age, name = 'Unknown', parent_permission: true)
    @id = Random.rand(1..9999)
    @name = name
    @age = age
    @parent_permission = parent_permission
    @rentals = []
    super()
  end
```